### PR TITLE
User registration and inserting to DB

### PR DESCRIPTION
**Pull request by Alekhyagnv**

### CODE REVIEW

File: bd/checkUserAlreadyExists.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+	"github.com/SaiKumarMalve/Gator-News/models"
+	"go.mongodb.org/mongo-driver/bson"
+)
+
+func CheckUserAlreadyExists(email string) (models.User, bool, string) {
+	ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
+	defer cancel()
+
+	db := MongoCN.Database(("Gator-News"))
```
Comment (SaiKumarMalve) [inline bd/checkUserAlreadyExists.go:15]: DB name is incorrect and also the collection name is all lower case letters

**Review by SaiKumarMalve — CHANGES_REQUESTED** [review]
Please make the necessary changes

**Review by SaiKumarMalve — APPROVED** [review]
Looks good to me. Submit the code

File: bd/checkUserAlreadyExists.go (hunk 1 above)
Comment (Alekhyagnv) [reply bd/checkUserAlreadyExists.go]: I have updated the code. Please review.

**Pull request merged**